external/vcm/vcm/catalog.yaml

```diff
@@ -142,11 +163,19 @@ sources:
         - u200
         - v200
         - w500
+        - q1000
+        - q850
+        - q500
+        - TMP850
+        - TMP500
+        - TMP200
+        - UGRD850
+        - UGRD200
     args:
       storage_options:
         project: 'vcm-ml'
         access: read_only
-      urlpath: "gs://vcm-ml-intermediate/2020-05-coarsen-c384-diagnostics/coarsen_diagnostics/atmos_8xdaily_C3072_to_C384.zarr"
+      urlpath: "gs://vcm-ml-intermediate/2020-05-coarsen-c384-diagnostics/coarsen_diagnostics/atmos_8xdaily_add_vars.zarr"
```

Comment:
I think it's confusing to change the url for a given catalog entry.

The prognostic run diagnostics allow having multiple catalog entries for a single "category" of diagnostics. So I would suggest just making a new zarr that only has the new derived variables, and then it will get merged with the existing one.

See:

https://github.com/VulcanClimateModeling/fv3net/blob/149b545c5dd633cae746f938e84fb82906623bc0/workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/config.py#L5-L17

and 

https://github.com/VulcanClimateModeling/fv3net/blob/149b545c5dd633cae746f938e84fb82906623bc0/workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/load_diagnostic_data.py#L21-L33

Reply:
Oh, neat, I didn't know it could do that.